### DISCUSSION TITLE
Duplicate test suite is creating everytime

### DISCUSSION THREAD

**Discussion by pooja-esper** [gurock/trcli#226] [Q&A] [answered]
Command am using :trcli -y -c "real-device/trcli-config.yml" parse_junit -f "reports_samsung_R58R72EMZRK/output-junit.xml"

XML file : `
<testsuites name="test suites root">
  <testsuite failures="0" errors="0" skipped="1" tests="1" time="3049" name="tests.LoginTests">
    <properties>
      <property name="setting1" value="True"/>
      <property name="setting2" value="value2"/>
    </properties>
    <testcase classname="tests.LoginTests" name="test_case_1" time="159">
      <skipped type="pytest.skip" message="Please skip">skipped by user</skipped>
    </testcase>
    <testcase classname="tests.LoginTests" name="test_case_2" time="650">
    </testcase>
    <testcase classname="tests.LoginTests" name="test_case_3" time="121">
      <failure type="pytest.failure" message="Fail due to...">failed due to...</failure>
    </testcase>
  </testsuite>
</testsuites>`

trcli-config.yml file:
host: https://xyz.testrail.io
project: CUSTOMER_SCENARIOS
username: xyz@xyz.io
password: xyz@12345
title: "test suites root"

Suite is already existed,but each time its creating new suite instead of just creating runs which leading our testrail project repo with large number of duplicate testsuites

**Comment by acuanico-tr-galt** [comment, marked as answer]
Hi, you may use the options --suite-id or --suite-name under parse_junit command.
